### ISSUE TITLE
Test Set of Heart Sound Physionet Challenge 2016

### ISSUE THREAD

**Issue by Link456** [MIT-LCP/physionet#135]
Hello,

I am writing my master thesis on heart sound analysis and want to include the Physionet Heart Sound Classification Challenge of 2016. I saw the data [here](https://physionet.org/content/challenge-2016/1.0.0/) includes the training data and a validation set, which is a subset of the training set. The authors mentioned in their paper that they are going to publish the test set, which was held back in order to use it for the challenge scoring, once the challenge is over.
However, I have not been able to find the test set anywhere. Has it been published and if not, is it possible to access the test set somehow in order to publish relevant results on the data set even after the actual challenge?
Any help is kindly appreciated.
